battleship.py

```diff
@@ -109,8 +109,16 @@ def checkShip(grid, ship):
 Parameters: 2D list of ints ; int
 Returns: 2D list of ints
 '''
-def addShips(grid, numShips):
-    return
+def addShips(grid, numships):
+    a=0

```

Comment:
Maintain a proper naming conventions for a variables.

Reply:
done with changing variable